HISTORY.md

```diff
@@ -44,6 +44,7 @@
 * Support dynamically change `delayed_write_rate` and `max_total_wal_size` options via SetDBOptions().
 * Introduce DB::DeleteRange for optimized deletion of large ranges of contiguous keys.
 * Support dynamically change `delayed_write_rate` option via SetDBOptions().
+* Support dynamically change `stats_dump_period_sec` option via SetDBOptions().
```

Comment:
Please add the changelog to Unreleased section, which will be included in next release.